src/engraving/compat/midi/compatmidirenderinternal.cpp

```diff
@@ -1096,6 +1099,9 @@ void CompatMidiRendererInternal::doRenderSpanners(EventsHolder& events, Spanner*
     int staff = static_cast<int>(s->staffIdx());
 
     if (s->isPedal()) {
+        auto conf = mu::modularity::ioc()->resolve<mu::iex::midi::IMidiImportExportConfiguration>("iex_midi");
+        MScore::pedalEventsMinTicks = conf ? conf->pedalEventsMinTicks() : (1);
```

Comment:
There got to be a better place for this, but where?

Reply:
I'd say: delete `MScore::pedalEventsMinTicks` altogether and always use the value from the configuration directly. 
Also, no need to use `mu::modularity::ioc()->resolve` here; instead, you can inject the configuration in the `CompatMidiRendererInternal` class.

Reply:
Done, thanks